R/agg_scale.R

```diff
@@ -57,6 +55,53 @@
 #' @return \[`data.table()`\] with `id_cols` and `value_cols` columns for
 #'   requested aggregates or with scaled values.
 #'
+#' @section Severity Arguments:
+#' **`missing_dt_severity`**:
+#'
+#' Check for missing levels of `col_stem`, the variable being aggregated or
+#' scaled over.
+#' 1. `stop`: throw error (this is the default).
+#' 2. `warning` or `message`: throw warning/message and continue with
+#' aggregation/scaling where possible.
+#' 3. `none`: don't throw error or warning, continue with aggregation/scaling
+#' where possible.
+#' 4. `skip`: skip this check and continue with aggregation/scaling.
```

Comment:
Ok, cool, so `skip` would allow for aggregation of, say, `[0,1,3,4]` to `0-4 = 0+1+3+4` by ignoring the fact that `2` is missing?

Reply:
Yup exactly